### PR TITLE
fix(cleanup): Log correct cluster when implicit setting is used

### LINKED ISSUE #1793: Snuba cleanup for sentry onpromise
### Environment

Sentry self-hosted 21.3.0 (based on docker-compose from here https://github.com/getsentry/onpremise/blob/21.3.0/docker-compose.yml)

### Steps to Reproduce

1) Setup all containers and up snuba-cleanup container
2) Check logs for snuba-cleanup: Every 5 minutes in log - `Dropped 0 partitions on None`
It looks like variable CLICKHOUSE_HOST is ignored here
https://github.com/getsentry/snuba/blob/41d7fe76aaf8a594e8f6e84015607dcde3f67ad4/snuba/cli/cleanup.py#L13
After manual run command in container - `snuba cleanup --clickhouse-host CLICKHOUSE_HOST_HERE --dry-run True`
i got `Dropped 0 partitions on 10.67.49.169`

### Expected Result

Pass variable https://github.com/getsentry/onpremise/blob/bdd2686021cfea07507bc07d2756ac34a775c680/docker-compose.yml#L44 into cleanup command

### Actual Result

variable is `None` instead of clickhouse host

I'am not sure, bug this or not.

### PR DESCRIPTION
**Pull request by BYK**
Fixes #1793.

### CODE REVIEW

**Comment by BYK** [comment]
Verified locally that this fixes the issue.

File: .vscode/settings.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
   "python.linting.flake8Enabled": true,
   "python.formatting.provider": "black",
   // https://github.com/DonJayamanne/pythonVSCode/issues/992
-  "python.pythonPath": ".venv/bin/python",
+  "python.pythonPath": "C:\\Users\\byk\\AppData\\Local\\Programs\\Python\\Python38\\python.exe",
```
Comment (evanh) [inline .vscode/settings.json:49]: This doesn't seem right.

Comment (BYK) [reply .vscode/settings.json:49]: Just because you don't use Windows and your username is not `byk` doesn't make it wrong :P

Comment (BYK) [reply .vscode/settings.json]: Fixed, thanks for the heads up! 😄

**Review by evanh — APPROVED** [review]

**Pull request merged**